src/index.ts

```diff
@@ -1,13 +1,44 @@
 import express from "express";
 import type { Express, Request, Response } from "express";
-
+// import type { Task } from "../DB/dbConfig";
+import prisma from "../DB/dbConfig";
 const app: Express = express();
-const port = parseInt(process.env.PORT ?? "3000");
+const port = parseInt(process.env.PORT ?? "7000");
 
-app.get("/", (req: Request, res: Response) => {
-  res.send("Hello World!");
+app.use(express.json());
+// test
+app.post("/tasks", async (req: Request, res: Response) => {
```

Comment:
Make use of separate route files for each module, all organized under one base. It makes code clear, scalable, and easy to manage.
Example:
app.use("/api/v1", appRoutes);
Your appRoutes should contain route file of every module

Reply:
Sure that was just for testing the database will setup in next branch and create a new PR

Reply:
Got it.